src/config/dynamic-theme-fixes.config

```diff
@@ -14871,6 +14871,18 @@ body,
 .searchTitle {
     background: none !important;
 }
+.sc-classic .header__navMenu > li > a {
+    border-right-color: #454545 !important;
+}
+.headerSearch__input {
+    background: ${#f1f4f6} !important;
+}
+.g-dark a {
+   color: #bbbbbb;
+}
+.g-dark a:hover {
+   color: #ffffff;
+}
```

Comment:
This seems to be a cosmetic fix and not actually fixing broken behavior, not justified to be added to the fixes.

Reply:
```suggestion
```